geode-core/src/main/java/org/apache/geode/internal/cache/AbstractRegionMap.java

```diff
@@ -1118,13 +1119,8 @@ public void txApplyDestroy(Object key, TransactionId txId, TXRmtEvent txEvent,
                     clearOccured /* Clear Conflciting with the operation */);
                 if (invokeCallbacks) {
```

Comment:
Similar pattern is used in multiple places. This could be moved to a separate function.